### PR TITLE
Clean dc he support final

### PR DESCRIPTION
**Pull request by tareqalayan**

### CODE REVIEW

**Comment by tareqalayan** [comment]
ci build please

File: roles/oVirt.datacenters/README.md [hunk 1]
```diff
@@ -3,6 +3,10 @@ oVirt Datacenters
 
 The `oVirt.datacenters` role is used to set up or cleanup oVirt datacenters.
 
+In the special case of cleaning up oVirt datacenter that contains a Hosted Engine VM
+the behavior of the role is different and is explained in more details in:
+[ovirt-datacenter-cleanup]: https://github.com/oVirt/ovirt-ansible/blob/master/roles/ovirt-datacenter-cleanup/README.md
```
Comment (petr-balogh) [inline roles/oVirt.datacenters/README.md]: I think that md file has different syntax for links, see https://github.com/tareqalayan/ovirt-ansible-infra/blob/4867e8a81407a742512abf3276dc07f072f88e95/roles/oVirt.datacenters/README.md
like this most likely?
[Contribution guidelines for this project](https://github.com/oVirt/ovirt-ansible/blob/master/roles/ovirt-datacenter-cleanup/README.md)

**Review by petr-balogh — APPROVED** [review]

File: roles/oVirt.datacenter-cleanup/tasks/find_hosted_engine_details.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  ovirt_vms_facts:
+    auth: "{{ ovirt_auth }}"
+    pattern: "datacenter={{ data_center_name }}"
+
+- name: Find HE info if HE
+  set_fact:
+    hosted_engine: "{{ ovirt_vms | selectattr(search_key, 'equalto', search_val) | list | count > 0 }}"
+  vars:
+    search_key: origin
+    search_val: managed_hosted_engine
```
Comment (machacekondra) [inline roles/oVirt.datacenter-cleanup/tasks/find_hosted_engine_details.yml:12]: Should we support also: https://github.com/oVirt/ovirt-engine/blob/06ce94db7eda2567eeb5471d899f9df901f12f15/backend/manager/modules/common/src/main/java/org/ovirt/engine/core/common/businessentities/OriginType.java#L15 ?

File: roles/oVirt.datacenter-cleanup/tasks/main.yml [hunk 3]
```diff
@@ -11,27 +14,29 @@
...
   include: storages_pre.yml
 
 - name: Find existing clusters in Datacenter
-  ovirt_clusters_facts:
+  ovirt_cluster_facts:
     auth: "{{ ovirt_auth }}"
     pattern: "datacenter={{ data_center_name }}"
 
+- name: Remove hosts except from the one hosting the engine VM
+  include: remove_hosts_not_hosting_engine_vm.yml
```
Comment (machacekondra) [inline roles/oVirt.datacenter-cleanup/tasks/main.yml:26]: include is depracated: https://docs.ansible.com/ansible/2.5/user_guide/playbooks_reuse_includes.html

**Comment by mwperina** [comment]
Kobi, is this PR still relevant? If so, could you please rebase and reverify?

@kobihk

**Comment by mnecas** [comment]
Closing, no longer relevant, if you want to continue with the PR please add it to ovirt-ansible-collection

**Pull request closed without merging**